operators/katib-controller/src/webhooks.yaml

```diff
@@ -63,9 +63,6 @@ webhooks:
         name: katib-controller
         namespace: kubeflow
         path: /mutate-pod
-    namespaceSelector:
-      matchLabels:
-        katib.kubeflow.org/metrics-collector-injection: enabled
```

Comment:
We shouldn't remove this `namespaceSelector`.
Otherwise, Katib Mutation Webhook will be executing in every Kubernetes namespace.